api-tests/package.json

```diff
@@ -7,7 +7,7 @@
     "test:awesome": "mocha specs/*/*.js --reporter mochawesome || true",
     "open:report": "open mochawesome-report/mochawesome.html",
     "test": "mocha specs/*/*.js",
-    "test-ci": "mocha specs/*/*.js --reporter mocha-junit-reporter"
+    "api-test-ci": "npm i && mocha --timeout 8000 --reporter mocha-junit-reporter --reporter-options mochaFile=./api-test-results.xml specs/practitioner-service/*.js || true"
```

Comment:
practitioner-service should be replace by *

to include other tests ?

Reply:
Had no role service to try that against, it's in compose now - great catch, I'll add it in